src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -19277,9 +19277,6 @@ static void F3(params object x, params object[] y)
                 // (13,20): error CS0225: The params parameter must be a single dimensional array
                 //     static void F3(params object x, params object[] y)
                 Diagnostic(ErrorCode.ERR_ParamsMustBeArray, "params").WithLocation(13, 20),
-                // (13,20): error CS0231: A params parameter must be the last parameter in a parameter list
-                //     static void F3(params object x, params object[] y)
-                Diagnostic(ErrorCode.ERR_ParamsLast, "params object x").WithLocation(13, 20),
```

Comment:
A few occurrences of `CS0231: A params parameter must be the last parameter in a parameter list` have been removed from tests. That's because checking for `params` being the last parameter was moved into `ReportParameterErrors`, as `else if`, so it's no longer reported if there are other `params`-related errors.

Reply:
I'd rather keep these errors if we can. They're not duplicates, and fixing an error just to get another error we could have reported the first time is not a great user experience.

Reply:
I agree it's not a good user experience, I just wanted to be consistent - the other parameter errors behave this way too since there is a chain of `else if`s:

https://github.com/dotnet/roslyn/blob/440b6fc370f0ddcea68b445e183f1383c0d1bc6e/src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs#L609-L670

Reply:
Anyway, changed: 883d02f096b95595227bc4ef59b6ddd54b0e1634